practice-two/src/style/sections/advantage.css

```diff
@@ -0,0 +1,189 @@
+.advantages {
+  background-color: var(--color-base);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.advantages__header {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 20px;
+}
+
+.advantages__header-text {
+  color: var(--color-secondary);
+}
+
+.advantages__description {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 140px;
+}
+
+.advantages__content {
+  display: grid;
+  width: 100%;
+  margin-top: 35px;
+  margin-bottom: 32px;
+  justify-content: space-around;
+  gap: 25px;
```

Comment:
Group the styles for display grid or other elements together. I mean include all similar styles in one place . E.g

```
.advantages__content {
  display: grid;
  justify-content: space-around;
  gap: 25px;
  width: 100%;
  margin-top: 35px;
  margin-bottom: 32px;
  ```